docker/kanister-elasticsearch/image/Dockerfile

```diff
@@ -1,12 +1,14 @@
 ARG TOOLS_IMAGE
 FROM ${TOOLS_IMAGE} AS TOOLS_IMAGE
 
-FROM alpine:3.11.3
+FROM debian:bullseye
 COPY --from=TOOLS_IMAGE /usr/local/bin/restic /usr/local/bin/restic
 COPY --from=TOOLS_IMAGE /usr/local/bin/kopia /usr/local/bin/kopia
 ADD kando /usr/local/bin/
 
-ADD docker/kanister-elasticsearch/image/esdump-setup.sh /esdump-setup.sh
-RUN chmod +x /esdump-setup.sh && sync && /esdump-setup.sh
+RUN apt update
+RUN apt install -y nodejs npm bash curl
```

Comment:
Was able to delete the shell script and directly call the install in the Dockerfile. Not sure why we used the shell script to begin with.